Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -19,81 +19,40 @@ function Get-AgentIPRange
 
     [hashtable] $IPRange = @{}
 
-    $sqlCmd = Get-Command -Name "SqlCmd.exe" -ErrorAction SilentlyContinue
-    if ($sqlCmd)
-    {
-        $sqlCmdArgs = "-S `"$serverName`" -U `"$sqlUsername`" -P `"$sqlPassword`" -Q `"select getdate()`""
+    $sqlCmd = Split-Path -Parent $MyInvocation.MyCommand.Path | Join-Path -ChildPath "sqlcmdfolder\SQLCMD.exe"
```

Comment:
No place holders .. make everything ready. When we have sqlcmd.exe uploaded we can check in.